.travis.yml

```diff
@@ -132,6 +132,28 @@ matrix:
      script:
        - docker run --rm -e PLAT=$PLAT -v `pwd`:/io $DOCKER_IMAGE $PRE_CMD /io/tools/build_wheels.sh
        - ls wheelhouse/
+   - os: linux
+     language: sh
+     env: MAKE_STEP=build_deb RUN_COMMAND=
```

Comment:
This sets RUN_COMMAND to empty and avoids running the gbp command in Docker during the Travis build. If it is what you wanted, fine as it works, just unsure.

Reply:
Yes, this is intentional. The deb build does not benefit from any (extra) layer of Docker